src/java.xml/share/classes/jdk/xml/internal/jdkcatalog/w3c/dtd/schema10/XMLSchema.dtd

```diff
@@ -239,36 +206,26 @@
             default            CDATA                  #IMPLIED
             fixed              CDATA                  #IMPLIED
             form               %formValues;           #IMPLIED
-            targetNamespace    %URIref;               #IMPLIED
             %elementAttrs;>
 <!-- type and ref are mutually exclusive.
      name and ref are mutually exclusive, one is required -->
 <!-- In the absence of type AND ref, type defaults to type of
-     substitutionGroup, if any, else xs:anyType, i.e. unconstrained -->
+     substitutionGroup, if any, else the ur-type, i.e. unconstrained -->
```

Comment:
ur-type does not appear in the original; is that intentional? 
(Ref https://www.w3.org/2009/XMLSchema/XMLSchema.dtd)